errai-bus/src/main/java/org/jboss/errai/bus/client/framework/transports/HttpPollingHandler.java

```diff
@@ -340,6 +342,18 @@ public boolean canWait() {
   public void handleProtocolExtension(final Message message) {
   }
 
+  private static void validateContentType(final Response response) {
+    int statusCode = response.getStatusCode();
+    // in case the response is OK (200), then the content type always has to be 'application/json'
+    if (statusCode == 200) {
```

Comment:
@mbarkley I am not sure about this. In case the statusCode is .e.g `404`, then content will of course not be `application/json`, but we still want to continue as the client side will try to recover from this (as it may just be a temporary network issue). We definitely only need to check some status code, but is 200 the only one?